src/main/resources/default/templates/biz/storage/list.html.pasta

```diff
@@ -205,5 +214,18 @@
                 modalElement.find("#select-file-modal-search-filter").focus();
             }).modal('show');
         }
+
+        function copyToClipboard(value) {
+            /*!
+             * inspired by OXOMI´s implementation of oxomi.copyToClipboard()
```

Comment:
I'd rather put that into a commit comment - the OXOMI repo is closed source anyway..